### ISSUE TITLE
display type aliases for mixed array items (e.g. anyOf)

### ISSUE THREAD

**Issue by ccwienk** [coveooss/json-schema-for-humans#60]
JSON-schema allows the definition of arrays whose items may be of different types (which themself can be again defined in the same schema). Currently, the generated schema browser will merely call them "Option 1", "Option 2", ... and display their resolved schema.

However, it would be more expressive to display the names through which they were defined.

Consider for example (simplified, of course):

```
definitions:
  customer:
   type: object
   properties: ['customer-id']
  employee:
    type: object
    properties: ['employee-id']

type: object
properties:
  people:
    type: array
    items:
     anyOf:
      - $ref: '#/definitions/customer'
      - $ref: '#/definitions/employee'
```

In this example, it would be much more expressive to name the available "options" `customer` and `employee`.

Please note that this example is of course simplified, and does certainly in this form not make too much sense. I derived it from an actual (I think reasonable) use-case.

Cheers
Christian

**Comment by dblanchette** [comment]
Hi and sorry for the delay in answering.

This is something that have been bothering me as well since the beginning. I think just extracting the last part of the ref name (after the last `/`) would be an acceptable solution for most use cases. What do you think?

I will look into it when I have some time. A PR is also welcome.

Have a nice day!

**Comment by dblanchette** [comment]
@ccwienk A solution to this issue has been released in version 0.22

Here is what the example you provided look like in this version:
![firefox_DqovawEr6t](https://user-images.githubusercontent.com/5759247/91889909-9dfd7000-ec5c-11ea-8c0e-c9f6bf13e533.png)

Don't hesitate if you have comments!

**Comment by ccwienk** [comment]
dear @dblanchette : thank you so much for implementing this - and sorry for my very delayed response :-)
